### PR TITLE
Issue #404 Adds RPG with one shot to nukieplanet

### PR DESCRIPTION
**Pull request by terezi-station**
## About the PR
Adds a RPG with one rocket to nukie planet

## Why / Balance
Issue #404
Allows a one time quick breach of walls

## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Added one time use RPG to nukie planet

### CODE REVIEW

File: Resources/Maps/Nonstations/nukieplanet.yml [hunk 1]
```diff
@@ -13611,7 +13613,7 @@ entities:
     - type: Transform
       pos: 14.341918,-16.877413
       parent: 104
-- proto: Telecrystal5 # Goobstation - Telecrystal rework
```
Comment (misandrie) [inline Resources/Maps/Nonstations/nukieplanet.yml:13614]: unrelated but lol

Comment (Aidenkrz) [reply Resources/Maps/Nonstations/nukieplanet.yml:13614]: related, why did you modify this lol

Comment (terezi-station) [reply Resources/Maps/Nonstations/nukieplanet.yml:13614]: idfk broe i havent mapped in 8 months idk why a buncha shit changed guessing its fine

Comment (misandrie) [reply Resources/Maps/Nonstations/nukieplanet.yml:13614]: > related, why did you modify this lol

Savemap automatically does this, commenting shit in map files its goofy

Comment (Aidenkrz) [reply Resources/Maps/Nonstations/nukieplanet.yml:13614]: that is goofy ash

**Comment by packmore** [comment]
nukies already start with a ton of free breaching bombs on their ship. this feels completely unneeded and it would most likely just result in nukies saving it to kill a bunch of people at once instead.

**Comment by Aidenkrz** [comment]
> nukies already start with a ton of free breaching bombs on their ship. this feels completely unneeded and it would most likely just result in nukies saving it to kill a bunch of people at once instead.

agreed.

**Comment by JoeHammad1844** [comment]
> nukies already start with a ton of free breaching bombs on their ship. this feels completely unneeded and it would most likely just result in nukies saving it to kill a bunch of people at once instead.

could just place the rocket launcher on the ship and remove a few C4's. the tradeoff is that C4 can destroy reinforced walls after 10 seconds (or instantly if you bother to set up a remote signaller), while it takes both shots with the RPG to destroy reinforced walls.

im intending it to be used against reinforced glass windows/regular walls, for purposes such as breaching fast into cargo or the bridge while you use C4 for more vital places like armory/vault

also i forgot to mention you get TWO shots with this (one is already in the launcher) but thats intended in the issue anyway as youll need a second to destroy the girder after you destroy a regular wall.

TLDR: rockets are NOT as powerful as C4

**Comment by JoeHammad1844** [comment]
also it would be funny for them to leave it on the ship and for the clown to use it against them

**Comment by packmore** [comment]
> TLDR: rockets are NOT as powerful as C4
against structures. against people, rockets are exceptionally strong and will insta-crit almost anybody outside of the other nukies. if somebody does not have specialized bomb protection, they will absolutely die to this. it's not the literal worst thing possible, but i'd really prefer just keeping the C4 considering nukies really don't need to be even more lethal.

**Comment by JoeHammad1844** [comment]
> against structures. against people, rockets are exceptionally strong and will insta-crit almost anybody outside of the other nukies. if somebody does not have specialized bomb protection, they will absolutely die to this. it's not the literal worst thing possible, but i'd really prefer just keeping the C4 considering nukies really don't need to be even more lethal.

fair point, but you only get two shots. i think its closer to a free trial for the china lake in that regard

**Comment by misandrie** [comment]
Free china is a no, you were banned from wizbuild repo for trying to pull similar things.

**Comment by JoeHammad1844** [comment]
> Free china is a no in any way possible, you were banned from wizbuild repo for trying to pull similar things.

actually i was banned for trying to add syndicate themed RCD sprites :godo:
anyway, no reason to bring that up

**Comment by Verbalase** [comment]
backflip
